src/main/java/tech/jhipster/lite/module/domain/gradleplugin/GradlePluginImports.java

```diff
@@ -0,0 +1,54 @@
+package tech.jhipster.lite.module.domain.gradleplugin;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.stream.Stream;
+import tech.jhipster.lite.shared.error.domain.Assert;
+
+public final class GradlePluginImports {
```

Comment:
I think we can remove the builder and just have:
```suggestion
public record GradlePluginImports(Collection<GradlePluginImport> imports) {

  public GradlePluginImports {
    Assert.field("imports", imports).notNull().noNullElement();
  }

  public void forEach(Consumer<GradlePluginImport> consumer) {
    Assert.notNull("consumer", consumer);

    imports.forEach(consumer);
  }

}
```